Task4.py

```diff
@@ -0,0 +1,19 @@
+"""
+Home task 4. If- statements.
+
+Task description:
+we have four values w,x,y,z
+write if-elif-else statement that will search minimum value and print smth aka "'y' is minimum value'
+advice use python debugger and different values to check your algorithm
+  """
+
+w, x, y, z = 100, 200, 40, 300
+
+if w < x and w < y and w < z:
```

Comment:
Algorithm looks as copy paste from internet. Just try to answer  to yourself if this is True or not )

Reply:
No, unfortunately, it was my idea, probably the bad one, but obvious for me and the first one to pop up. 
So what is wrong with it? Difficult to comprehend? 
Will think about other solutions then.